README.md

```diff
@@ -1 +1,136 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/">HTML</a></li>
+    <li><a href="https://www.w3.org/TR/CSS/#css">CSS</a></li>
+    <li><a href="https://www.w3.org/TR/JS/#js">JS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Set up the linter for the project 👌
+- Add the .gitignore file🚀
+- Add the styling
+- Add input name and scores forms
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+
+1. Computer
+2. Internet
+3. Web Browser
+4. Git
+5. Code Editor
+
+### Setup
+
+Clone this repository to your desired folder Enter the following commands in the terminal:
+
+```sh
+  git clone git@github.com:efmomoh/leaderboard.git
+  cd leaderboard
+```
+
+### Install
+
+Install this project with the following command in the terminal:
+
+```sh
+ npm install 
+
+```
+
+### Usage
+
+To run the project,
+
+```sh
+  Open the browser and enter the address of the index.html file from your computer/server
+```
+
+### Run tests
+
+```sh
+   1. npx hint.
+   2. npx stylelint "**/*.{css,scss}"
+```
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+ - [Live Demo Link](https://efmomoh.github.io/leaderboard/) -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Enssah**
+
+- GitHub: [@githubhandle](https://github.com/efmomoh)
+- Twitter: [@twitterhandle](https://twitter.com/@efmomoh)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/efmomoh?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3BQI%2F5GWZxS063VqRg2rilyg%3D%3D)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+- basic page structure.
+- send and receive data from API.
+- final touches.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 contributing <a name="contributing"></a>
+
+You are welcome, and click <a href="https://github.com/efmomoh/leaderboard/issues">here</a> for contribution, issues, and feature request 🙏
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star and follow me on GitHub LinkedIn and Twitter
+, you are also welcome with any suggestions🙏
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my Microverse codding, partners, morning sessions and my standup teams for helping and encouraging me to do this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] The link to the MIT license is not working as expected, please make sure it is working as displayed in the Microverse template.

Reply:
Thank you for the feedback. I will fix it.